README.md

```diff
@@ -9,20 +9,20 @@
 
 Firewood is an embedded key-value store, optimized to store recent Merkleized blockchain
 state with minimal overhead. Firewood is implemented from the ground up to directly
-store trie nodes on-disk. Unlike most of state management approaches in the field,
+store trie nodes on-disk. Unlike most state management approaches in the field,
 it is not built on top of a generic KV store such as LevelDB/RocksDB. Firewood, like a
 B+-tree based database, directly uses the trie structure as the index on-disk. Thus,
 there is no additional “emulation” of the logical trie to flatten out the data structure
 to feed into the underlying database that is unaware of the data being stored. The convenient
 byproduct of this approach is that iteration is still fast (for serving state sync queries)
 but compaction is not required to maintain the index. Firewood was first conceived to provide
 a very fast storage layer for the EVM but could be used on any blockchain that
-requires authenticated state.
+requires an authenticated state.
 
 Firewood only attempts to store the latest state on-disk and will actively clean up
-unused state when state diffs are committed. To avoid reference counting trie nodes,
+unused states when state diffs are committed. To avoid reference counting trie nodes,
```

Comment:
I think replacing `states` with `data` would be even more clear. Otherwise this PR LGTM. Thanks!